app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -72,6 +97,18 @@ def rrd_special_issue_set?
     end
   end
 
+  def diagnostic_codes
+    form_disabilities = form.dig('form526', 'form526', 'disabilities')
+    form_disabilities.map { |disability| disability['diagnosticCode'] }
+  end
+
+  # TODO: extract into constants code
+  def forward_to_mas?
+    RapidReadyForDecision::Constants::MAS_DISABILITIES.any? do |i|
+      diagnostic_codes.include?(i)
+    end
+  end
+
```

Comment:
makes sense, i could mimic how `.rrd_applicable?` gates stuff

Reply:
Here's an example of where that logic is currently, they could be used as a basis for adding the logic to `forward_to_mas?`
`SidekiqJobSelector.disability_increase?` and `RrdSpecialIssueManager.included_disability_increase?`

Reply:
This was addressed in method `forward_to_mas?`

```ruby
  def forward_to_mas?
    # only use the first diagnostic code because we can only support single-issue claims
    diagnostic_codes.size == 1 && RapidReadyForDecision::Constants::MAS_DISABILITIES.include?(diagnostic_codes.first)
  end
```